README.md

```diff
@@ -2,10 +2,59 @@
 
 ![License: Apache 2.0](https://img.shields.io/badge/license-Apache%202.0-blue)
 
-# Install latest release
+Kubernetes Dashboard is an open source extension for Podman Desktop to monitor with Kubernetes clusters.
+
+The Kubernetes Dashboard detects your Kubeconfig file, and connects to the current Kubernetes context.
+Any change of Kubernetes context (either from Podman Desktop or by editing the Kubeconfig file with another tool) 
+is detected by the dashboard, which disconnects from the previous current context and connects to the new one.
+
+A Dashboard page provides a synthetic view of the main resources present in the Kubernetes context: 
+- non-namespaced resources: nodes and namespaces, 
+- namespaced resources: workloads (deployments, pods, jobs, cronjobs), services (services, ingresses, routes) and configuration and storage (persistent volume claims, configmaps, secrets).
+
+You can access from the dashboard, or from the menu, the list of resources of a specific kind (for example, the list of Pods).
+
+From this list, you have access to the details for a specifc resource, including:
+- a summary of the resource (extracted from the metadata, spec and status of the resource),
+- a raw JSON representation of the resource,
+- a utility to patch the resource definition, using [strategic merge patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/).
+
+For Pods, you also have access to:
+- the logs of the containers running in the pod.
+
+You can switch the namespace you want to explore from the Dashboard page or from any list of namespaced resources. The extension will disconnect from the previous namespace and connect to the new one.
+
+# Installation
+
+## Install latest release
 
 Install the custom extension `ghcr.io/podman-desktop/podman-desktop-extension-kubernetes-dashboard:latest` to try the latest extension.
 
-# Development version
+## Development version
 
 Install the custom extension `ghcr.io/podman-desktop/podman-desktop-extension-kubernetes-dashboard:next` to try the extension published after each commit.
+
+# Contributing
+
+First clone the current directory on your disk, then run `pnpm i` in the root directory of the sources, then awtch the changes in the `packages/webview` directory:
+
+```
+$ git clone git@github.com:podman-desktop/podman-desktop.git
```

Comment:
I would maybe add first the way without cloning podman desktop, just using a binary of podman desktop

Reply:
Ah my bad, this part is about cloning the sources of the extension, not podman desktop. I'll change